python/tvm/auto_scheduler/compute_dag.py

```diff
@@ -81,12 +81,15 @@ def apply_steps_from_state(self, state):
         state : Union[State, StateObject]
             The state from which we get transform steps.
 
+        layout_rewrite: Bool
+            Rewrite the layout of placeholder.
```

Comment:
This description doesnt add much beyond the variable name. What's the benefit of doing this? Maybe add a sentence describing when you'd want to set this.